Shared/Samples/Style point with picture marker symbols/StylePointWithPictureMarkerSymbolsView.swift

```diff
@@ -0,0 +1,86 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct StylePointWithPictureMarkerSymbolsView: View {
+    /// A graphics overlay to hold the picture marker symbol graphics.
+    @State private var graphicsOverlay: GraphicsOverlay = {
+        let graphics = [
+            makePictureMarkerSymbolFromImage(),
+            makePictureMarkerSymbolFromURL()
+        ]
+        return GraphicsOverlay(graphics: graphics)
+    }()
+    
+    /// A map with topographic basemap and centered on Harman's Cross in England.
+    @State private var map: Map = {
+        let map = Map(basemapStyle: .arcGISTopographic)
+        map.initialViewpoint = Viewpoint(
+            center: Point(x: -225166.5, y: 6551249, spatialReference: .webMercator),
+            scale: 1e5
+        )
+        return map
+    }()
+    
+    /// Creates a picture marker symbol from an image in the project assets.
+    /// - Returns: A picture marker symbol
+    private static func makePictureMarkerSymbolFromImage() -> Graphic {
+        let imageName = "PinBlueStar"
+        
+        // Create pin symbol using the image.
+        let pinSymbol = PictureMarkerSymbol(image: UIImage(named: imageName)!)
+        
+        // Change offsets, so the symbol aligns properly to the point.
+        pinSymbol.offsetY = pinSymbol.image!.size.height / 2
```

Comment:
You should probably at least mention the fact that `PictureMarkerSymbol` is `Loadable`, as this same code, if it was used in `makePictureMarkerSymbolFromURL()` would crash, because `image` is `nil` until loaded. Or maybe an `if let image = pinSymbol.image`.

Reply:
I added a guard statement in both the makePictureMarkerSymbolFromImage() and makePictureMarkerSymbolFromURL() methods. Let me know if that is what you were thinking or if you want something different.

Reply:
Since the user can overcome this limitation by simply calling `load()` on the symbol, I'd prefer not to explicitly check to see if the symbol is loaded (even though `load()` is `async throws` and not so easy to implement here).

I added a suggestion below, instead of the `guard` you added.